src/mutations/itemFavorite.ts

```diff
@@ -0,0 +1,50 @@
+import { QueryClient, useMutation } from 'react-query';
+
+import { UUID } from '@graasp/sdk';
+
+import * as Api from '../api';
+import { FAVORITE_ITEMS_KEY, MUTATION_KEYS } from '../config/keys';
+import { addFavoriteItemRoutine, deleteFavoriteItemRoutine } from '../routines';
+import { QueryClientConfig } from '../types';
+
+export default (queryClient: QueryClient, queryConfig: QueryClientConfig) => {
+  const { notifier } = queryConfig;
+
+  queryClient.setMutationDefaults(MUTATION_KEYS.ADD_FAVORITE_ITEM, {
```

Comment:
Do you think you could refactor this file given this new format? https://github.com/graasp/graasp-query-client/blob/main/src/mutations/itemTag.ts